content/python/concepts/built-in-functions/terms/divmod/divmod.md

```diff
@@ -0,0 +1,62 @@
+---
+Title: 'divmod()'
+Description: 'Returns the quotient and remainder of the division of two numbers.'
+Subjects:
+
+    'Computer Science'
+    'Mathematics'
+    Tags:
+    'Functions'
+    'Math'
+    'Numbers'
+    CatalogContent:
+    'learn-python-3'
+    'paths/computer-science'
+    'paths/mathematics'
+---
+
+Returns the quotient and remainder of the division of two numbers.
+
+## Syntax
+
+```py
+divmod(x, y)
+```
+
+## Example 1
+
+Use `divmod()` to calculate the quotient and remainder of dividing `10` by `3`:
+
+```codebyte/python
+result = divmod(10, 3)
+
+print(result)  # Output: (3, 1)
+```
+
+In this example, the quotient of 10 divided by 3 is 3, and the remainder is 1.
+
+
+## Example 2
+
+Use `divmod()` to calculate the quotient and remainder of dividing `-15` by `4`:
+
+```codebyte/python
```

Comment:
you can remove the output comment because the codebyte is an interactive version, when a user hits 'run' the output will be displayed on the page